rma_delivery/models/rma_operation.py

```diff
@@ -8,6 +8,16 @@ class RmaOperation(models.Model):
 
     _inherit = "rma.operation"
 
-    default_carrier_id = fields.Many2one(
-        comodel_name="delivery.carrier", string="Default carrier", required=False
+    default_carrier_id_in = fields.Many2one(
```

Comment:
Maybe with change of field name, We should add a migration script to put current value in two new fields

Reply:
I agree. Also, now that I see it, I would name the fields `default_carrier_in_id` and `default_carrier_out_id` to respect naming [conventions](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields).